assets/main.css

```diff
@@ -1181,4 +1181,27 @@ tbody {
     font-size: 13px;
     padding: 17px 5px 0;
   }
+
+  .popup .popuptext::after {
+    content: '';
+    position: absolute;
+    bottom: -11px;
+    left: 125px;
+    background-color: #fff;
+    display: block;
+    width: 20px;
+    height: 20px;
+    transform: rotate(45deg);
+    border-radius: 0 0 5px;
+    border-right: 1px solid #d7d7db;
+    border-bottom: 1px solid #d7d7db;
+    border-left: 1px solid #fff;
+    border-top: 1px solid #fff;
+  }
```

Comment:
It shouldn't be necessary to redefine all these css attributes here. We should only set the ones that need to be overridden or added.

Reply:
Yeah that's true, I'll make the required changes